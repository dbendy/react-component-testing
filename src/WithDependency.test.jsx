/* eslint-env jest */

import React from 'react';
import renderer from 'react-test-renderer';
import ComponentWithDependency from './WithDependency';

jest.mock('./LibComponents', () => 'LibraryButton');

describe('ComponentWithDependency', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<ComponentWithDependency />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
