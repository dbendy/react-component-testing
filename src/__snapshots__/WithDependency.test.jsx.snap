// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentWithDependency matches snapshot 1`] = `
<div>
  <h1>
    Unwanted popup!  Click ok to make this go away.
  </h1>
  <LibraryButton
    text="Ok"
  />
</div>
`;
